wavelength-plugins/factions.js

```diff
@@ -529,6 +529,8 @@ exports.commands = {
 
 			factions[factionid].invites.splice(factions[factionid].invites.indexOf(user.userid), 1);
 			write();
+
+			user.popup("You've declined the invitation to join " + factions[factionid].name + ".");
```

Comment:
It would probably be best to reply in PMs since the original message was in a PM box as well (if this is possible, id have to review code for it)

Reply:
Just a suggestion, I would register the username WL Faction, and change it to something like `this.send("|pm| WL Faction" + "|" + user.getIdentity() + "|" + "You've declined the invitation to join " + factions[factionid].name + ".");`

Reply:
However, if we did it like that we would need to change the rest of the messages as well for consistency.